packages/components/src/filters/advanced/number-filter.js

```diff
@@ -0,0 +1,184 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { Component, Fragment } from '@wordpress/element';
+import { SelectControl, TextControl } from '@wordpress/components';
+import { get, find, partial } from 'lodash';
+import interpolateComponents from 'interpolate-components';
+import classnames from 'classnames';
+import { _x } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import TextControlWithAffixes from '../../text-control-with-affixes';
+import { formatCurrency } from '../../../../currency/src';
```

Comment:
`formatCurrency` is not in this package, so you should use `@woocommerce/currency` here. The reason for relative imports is to not create circular dependencies inside a single package– so files in components should be imported using relative paths, but other packages (anything top-level in the `packages` dir) should still use the `@woocommerce/<name>` syntax.

Also, if you've added any new package imports, those need to be added to the local package.json (ex `packages/components/package.json`). I don't think you have, but for future work :)